.github/workflows/test-gb-25.yml

```diff
@@ -179,7 +179,7 @@ jobs:
         timeout-minutes: 60
         run: |
           export XLA_FLAGS='--xla_dump_to=${{ env.GB25_DIR }}/xla_dump'
-          timeout --signal=TERM --verbose 59m mpiexecjl -np 1 julia --color=yes --project -O0 sharding/sharded_baroclinic_instability_simulation_run.jl
+          timeout --signal=TERM --verbose 59m mpiexecjl -np 1 julia --vmodule=spmd_partitioner*=10 --color=yes --project -O0 sharding/sharded_baroclinic_instability_simulation_run.jl
```

Comment:
`--vmodule` isn't a Julia flag.  Is there a way to achieve this via `XLA_FLAGS` instead?